mithril-signer/src/services/epoch_service.rs

```diff
@@ -419,11 +509,106 @@ mod tests {
         assert!(!service
             .is_signer_included_in_current_stake_distribution(
                 party_id,
-                protocol_initializer_not_included
+                &protocol_initializer_not_included
             )
             .unwrap());
     }
 
+    #[test]
+    fn signer_cant_sign_if_no_protocol_initializer_are_stored() {
```

Comment:
Those 4 tests about protocol initializer may be into a sub-module.
We also may have `test_protocol_initializer_is_available_after_register_epoch_settings_call_if_in_store()` in this module

Reply:
I grouped the four tests together under a 'can_signer_sign_current_epoch' umbrella, but since this name doesn't fit the last test that you suggested to move I kept it where it was.